waiter/src/waiter/main.clj

```diff
@@ -67,9 +72,12 @@
    :http-server (pc/fnk [[:routines waiter-request?-fn websocket-request-authenticator]
                          [:settings host port]
                          handlers] ; Insist that all systems are running before we start server
-                  (server/run-jetty {:ring-handler (consume-request-stream (core/ring-handler-factory waiter-request?-fn handlers))
+                        (server/run-jetty {:ring-handler (-> (core/ring-handler-factory waiter-request?-fn handlers)
+                                                             consume-request-stream
+                                                             wrap-request-cookies)
                                      :websocket-acceptor websocket-request-authenticator
-                                     :websocket-handler (core/websocket-handler-factory handlers)
+                                     :websocket-handler (-> (core/websocket-handler-factory handlers)
+                                                            wrap-request-cookies)
```

Comment:
I found this because all of the other integration tests broke due to the x-waiter-auth cookie being dropped. I don't think we send any cookies to the backend for web socket requests, so is there another type of test you'd want?